euphonic/readers/phonopy.py

```diff
@@ -105,24 +114,28 @@ def _extract_phonon_data_yaml(filename):
             [atom['symbol'] for atom in phonon_data['points']])
     except KeyError:
         pass
-
     return data_dict
 
 
-def _extract_phonon_data_hdf5(filename):
+def _extract_phonon_data_hdf5(filename: str,
+                              read_eigenvectors: Optional[bool] = True
```

Comment:
`Optional[bool]` means that the value can be `True`, `False` or `None` but I don't think that's correct? For a `bool` with a default value I think we just want `read_eigenvectors: bool = True`.

Or we could do a similar thing to `use_c` and have `None` mean "try" and `True` raise an error if eigenvectors unavailable. I don't think it's really necessary; it's easy enough to check `if data.get('eigenvectors')`.

Reply:
Agreed, have removed `Optional` in all places where it isn't applicable